<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #000000; color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid; border-width: 1px; border-color: red;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="eth0_textedit">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>539</y>
       <width>639</width>
       <height>179</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px; border-color: #ffffff; border-style: solid;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>eth0</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="wlan0_textedit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>539</y>
       <width>639</width>
       <height>179</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px; border-color: #ffffff; border-style: solid;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>wlan0</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ntpstat_textedit">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>438</y>
       <width>639</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px; border-color: #ffffff; border-style: solid;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>ntpstat</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="uptime_textedit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>438</y>
       <width>639</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px; border-color: #ffffff; border-style: solid;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>w</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_5">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>337</y>
       <width>639</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px; border-color: #ffffff; border-style: solid;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_6">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>337</y>
       <width>639</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px; border-color: #ffffff; border-style: solid;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>1278</width>
      <height>119</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.SF Compact Display</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid; border-color #ffffff;</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ip_label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>120</y>
      <width>1278</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.SF Compact Display</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid; border-color #ffffff;</string>
    </property>
    <property name="text">
     <string>ip</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hwaddr_label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>180</y>
      <width>1278</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.SF Compact Display</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid; border-color #ffffff;</string>
    </property>
    <property name="text">
     <string>hwaddr</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="internet_label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>240</y>
      <width>1278</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.SF Compact Display</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid; border-color #ffffff;</string>
    </property>
    <property name="text">
     <string>internet</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
